src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -13,19 +13,27 @@
     @Nullable
     private Integer recommendedLength;
 
+    private Boolean shouldAllowRichText;
+
     public FeedbackTextQuestionDetails() {
         super(FeedbackQuestionType.TEXT);
         recommendedLength = null;
+        shouldAllowRichText = true;
     }
 
     public FeedbackTextQuestionDetails(String questionText) {
         super(FeedbackQuestionType.TEXT, questionText);
         recommendedLength = null;
+        shouldAllowRichText = true;
     }
 
     @Override
     public boolean shouldChangesRequireResponseDeletion(FeedbackQuestionDetails newDetails) {
-        return false;
+        assert newDetails instanceof FeedbackTextQuestionDetails;
```

Comment:
Should use `Assumption` (search on codebase for example) class as `-ea` is turned off in GAE.